src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -62,10 +62,7 @@
   --#{$notification-drawer}__list-item-header-icon--MarginInlineEnd: var(--pf-t--global--spacer--sm);
 
   // List item header title
-  --#{$notification-drawer}__list-item-header-title--FontFamily: var(--pf-t--global--font--family--heading);
-  --#{$notification-drawer}__list-item-header-title--FontSize: var(--pf-t--global--font--size--heading--xs);
-  --#{$notification-drawer}__list-item-header-title--FontWeight: var(--pf-t--global--font--weight--heading--bold);
-  --#{$notification-drawer}__list-item-header-title--LineHeight: var(--pf-t--global--font--line-height--heading);
```

Comment:
looks like we removed the `__list-item-header-title` (title in an individual notification item) font vars, then updated properties for `__header-title`. Was that intentional, or did you mean to make property updates to this block? https://github.com/patternfly/patternfly/blob/b12c46a93ee106e5e77c98e049760209089d8deb/src/patternfly/components/NotificationDrawer/notification-drawer.scss#L228-L232